src/Salt/SiteBundle/Entity/CommentRepository.php

```diff
@@ -26,7 +26,14 @@ public function addComment($itemType, $itemId, User $user, string $content, $par
         $comment = new Comment();
```

Comment:
The annotation still shows that `$itemId` is an integer (as does the name), but it is not.